llvm/include/llvm/MC/TargetRegistry.h

```diff
@@ -453,14 +453,14 @@ class Target {
   /// either the target triple from the module, or the target triple of the
   /// host if that does not exist.
   TargetMachine *createTargetMachine(
-      StringRef TT, StringRef CPU, StringRef Features,
```

Comment:
Can we keep a StringRef overload, at least for a while? As-is this is going to break a lot of downstream builds

Reply:
Restored StringRef overload with deprecation.